qualification/antenna_channelised_voltage/test_linearity.py

```diff
@@ -42,11 +42,11 @@ async def test_linearity(
     cw_scales = [0.5**i for i in range(10)]
     pdf_report.detail(f"CW scales: {cw_scales}")
 
-    pdf_report.step("Select a channel and compute the channel center frequency for the D-sim.")
-    sel_chan_center = receiver.n_chans // 3
-    channel_frequency = sel_chan_center * (receiver.bandwidth / receiver.n_chans)
+    pdf_report.step("Select a channel and compute the channel centre frequency for the D-sim.")
+    sel_chan_centre = receiver.n_chans // 3
+    channel_frequency = sel_chan_centre * (receiver.bandwidth / receiver.n_chans)
```

Comment:
Since we're aiming for consistency, I'd suggest renaming to `sel_channel_centXX` so that we're locally consistent about "channel" versus "chan".